R/vulnerability/disasters-emergencies/england/access-services.R

```diff
@@ -0,0 +1,110 @@
+# Load packages 
+library(tidyverse)
+library(readxl)
+library(demographr)
+
+source("https://raw.githubusercontent.com/britishredcrosssociety/resilience-index/main/R/utils.R") # for download_file() & calculate_extent()
```

Comment:
Ah it was because the new `calculate_extent()` function was on master and not this branch. Sorry I should have rebased instead of this. Will fix once merged.

Reply:
No problem :)